macros/read_group_macros.xml

```diff
@@ -0,0 +1,278 @@
+<macros>
+    <!-- Import this at the top of your command block and then
+         define rg_auto_name. -->
+    <token name="@define_read_group_helpers@">
+#import re
+#def identifier_or_name($input1)
+#if hasattr($input1, 'element_identifier')
+#return $input1.element_identifier
+#else
+#return $input1.name.rstrip('.gz').rstrip('.fastq').rstrip('.fq')
```

Comment:
@jmchilton can we use indentation for cheetah code? This would make it so much easier to read.